src/utils/sidecar.js

```diff
@@ -4,8 +4,19 @@ export const openSidecar = async (program, type, data) => {
     const sidecar = Command.sidecar(program);
 
     const promiseDataReceived = new Promise((resolve, reject) => {
-        sidecar.stdout.on('data', e => resolve(e));
-        sidecar.stderr.on('data', e => reject(e));
+        // Listen for output
+        const stdout = [];
+        sidecar.stdout.on('data', e => stdout.push(e));
+        const stderr = [];
+        sidecar.stderr.on('data', e => stderr.push(e));
+
+        // Report output on exit
+        sidecar.on('close', () => {
+            if (stderr.length > 0)
+                reject(stderr.join('\n'));
+            else
+                resolve(stdout.join('\n'));
+        });
```

Comment:
Good idea. Though I'd resolve/reject with the array instead of making it a string

Reply:
Should be fixed with latest commit